bindings/cs/rl.net/Configuration.cs

```diff
@@ -27,5 +33,24 @@ public static bool TryLoadConfigurationFromJson(string json, out Configuration c
             int result = LoadConfigurationFromJson(json.Length, json, config.NativeHandle, apiStatus.ToNativeHandleOrNullptr());
             return result == NativeMethods.SuccessStatus;
         }
+
+        public string this[string key]
+        {
+            get
+            {
+                IntPtr result = ConfigurationGet(this.NativeHandle, key, string.Empty);
+                if (result == IntPtr.Zero)
+                {
+                    // TODO: Should we use string.Empty?
```

Comment:
>// TODO: Should we use string.Empty? [](start = 20, length = 36)

I think we should at least be consistent. If the default value of `ConfigurationGet` is `string.Empty` then I would suggest returning the same value here.